src/controllers/info.ts

```diff
@@ -18,6 +19,9 @@ export default function (server: Hapi.Server, deps: Injector) {
     const fullMem = podManager.getMemoryUsed() * (2 ** 20) / os.totalmem() >= config.maxMemoryFraction
     const infoResp: HostInfo = {
       fullMem,
+      serverUptime: os.uptime(),
+      serviceUptime: process.uptime(),
+      avgLoad: os.loadavg()[0],
```

Comment:
This might be a little risky to expose because it could be used to target easy DoS victims

Reply:
Only the load - uptime should be fine

Reply:
Agree, load + uptime is fine. This is information an attacker could already get by monitoring uptime and response times.